### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by Geckool** [400plus/400plus#458]
Hello, Firmware up to date, card, formatted in the camera. Made bootable under Linux (Ubuntu) according to the procedure in your tutorial. Hack copied to the root of the card. When I put it back in the device the blue light does not come on and the hack does not work. Thanks in advance.

**Comment by eduperez** [comment]
Did you also follow the steps at "Preparing the camera"?

**Comment by Geckool** [comment]
Oops sorry, I missed this step, however I did not find testfir.fir for version Version 20160404-10, so I used the one from Version 20160404-09. Tell me if it is OK otherwise where I can find the one for the latest version. In any case thank you very much for your excellent work and your great responsiveness. Best regards.

**Comment by eduperez** [comment]
Yes, there is just one version of TESTFIR.FIR: it's only used to toggle a
flag inside the camera.

**Comment by Geckool** [comment]
A new problem, everything seems to work normally but after a while, either a "Settings" menu with "ISO Settings" on each line or empty lines with settings, 0, 0, 45. To put things back in order, you have to delete all the files on the card and copy the 400PLUS folder and Autoexec

**Comment by Geckool** [comment]
![IMG20240709012343 (1)](https://github.com/400plus/400plus/assets/175014218/574aca3d-0d68-4a31-b03f-0ee093f6f75d)
That's what happens
